ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -31,34 +31,74 @@
+{%      for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
+{%          if ethernet_interfaces[ethernet_interface].isis_enable is arista.avd.defined %}
+{%              set row_isis_instance = ethernet_interfaces[ethernet_interface].isis_enable %}
+{%              set row_isis_metric = ethernet_interfaces[ethernet_interface].isis_metric | arista.avd.default('-') %}
+{%              if ethernet_interfaces[ethernet_interface].isis_network_point_to_point is arista.avd.defined(true) %}
+{%                 set row_intf_mode = "point-to-point" %}
+{%              elif ethernet_interfaces[ethernet_interface].isis_passive is arista.avd.defined(true) %}
+{%                 set row_intf_mode = "passive" %}
+{%              else %}
+{%                 set row_intf_mode = "-" %}
 {%              endif %}
-{%          endfor %}
-{%      endif %}
+| {{  ethernet_interface }} | {{ row_isis_instance }} | {{ row_isis_metric }} | {{ row_intf_mode }} |
+{%          endif %}
+{%      endfor %}
 {# ------- #}
 {# vlan interfaces #}
 {# ------- #}
-{%      if vlan_interfaces is defined and vlan_interfaces is not none %}
-{%          for vlan_interface in vlan_interfaces | arista.avd.natural_sort %}
-{%              if vlan_interfaces[vlan_interface].isis_enable is defined and vlan_interfaces[vlan_interface].isis_enable is not none %}
-| {{ vlan_interface }} | {{ vlan_interfaces[vlan_interface].isis_enable }} |  {{ vlan_interfaces[vlan_interface].isis_metric | default('-') }} | {% if vlan_interfaces[vlan_interface].isis_network_point_to_point is defined and vlan_interfaces[vlan_interface].isis_network_point_to_point is not none %} point-to-point {% elif vlan_interfaces[vlan_interface].isis_passive is defined and vlan_interfaces[vlan_interface].isis_passive is not none %} passive {% else %} - {% endif %}|
+{%      for vlan_interface in vlan_interfaces | arista.avd.natural_sort %}
+{%          if vlan_interfaces[vlan_interface].isis_enable is arista.avd.defined %}
+{%              set row_isis_instance = vlan_interfaces[vlan_interface].isis_enable %}
+{%              set row_isis_metric = vlan_interfaces[vlan_interface].isis_metric | arista.avd.default('-') %}
+{%              if vlan_interfaces[vlan_interface].isis_network_point_to_point is arista.avd.defined(true) %}
+{%                 set row_intf_mode = "point-to-point" %}
+{%              elif vlan_interfaces[vlan_interface].isis_passive is arista.avd.defined(true) %}
+{%                 set row_intf_mode = "passive" %}
+{%              else %}
+{%                 set row_intf_mode = "-" %}
 {%              endif %}
-{%          endfor %}
-{%      endif %}
+| {{ vlan_interface }} | {{ row_isis_instance }} |  {{ row_isis_metric }} | {{ row_intf_mode }} |
```

Comment:
Found an extra space here ```|  {{ row_isis_metric }} |``` which should have been one space only.

Reply:
Fixed.